release-tools/check-crate-version/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "check-crate-version"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
```

Comment:
can you add a license?  this should be apache2

Reply:
Done